src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -107,6 +144,17 @@ public override Type ExpectedClrType
             get { return typeof(TStructuralType); }
         }
 
+        /// <summary>
+        /// Gets the enum type of <see cref="EdmDeltaEntityKind"/>.
+        /// </summary>
+        public EdmDeltaEntityKind DeltaKind { get; set; }
+
+        /// <inheritdoc />
+        public IODataInstanceAnnotationContainer TransientInstanceAnnotationContainer { get; set; }
+
+        /// <inheritdoc />
+        public IODataInstanceAnnotationContainer PersistentInstanceAnnotationsContainer { get; set; }
```

Comment:
>set [](start = 95, length = 3)

Any reason to set these once the Delta<TStructuralType> is created? Especially since the PersistentInstanceAnnotationContainer should just redirect to the instance annotation container on the <TStructuralType> (if it exists) I wouldn't think this would be settable, and I don't know why a developer would want to set the TransientInstanceAnnotationContainer (although we could add the setter later if we found scenarios).